constraints/strongStart.js

```diff
@@ -1,6 +1,36 @@
+function seekCatLeftEdge(node, cat, id){
```

Comment:
I think this and searchDownLeft have been superceded by catDomsIdAtLeftEdge and its helper, hasIdAtLeftEdge, no? Looks like you forgot to delete these, or some kind of versioning weirdness happened.

Reply:
Haha, not sure how that happened